tests/test_metrics.py

```diff
@@ -0,0 +1,67 @@
+from nose.tools import (assert_less_equal, assert_less, assert_greater_equal,
+                        assert_greater,
+                        assert_equal)
+
+import numpy as np
+np.random.seed(42)
```

Comment:
Imports first

Reply:
I know the rule, but I always ignore it when it comes to random seeds. I had bad experience with setting the seed after all imports (other module might use it at import stage)

Reply:
I prefer setting seed right where using them, so locally, not in a global context. Whichever you prefer is also fine by me.

Reply:
Let's set them locally then